pkg/resource/model_package/testdata/test_suite.yaml

```diff
@@ -1,17 +1,251 @@
-tests:
-  - name: "Unversioned model package create tests"
-    description: "Part of unversioned model package CRD tests."
+  tests:
+  - name: "Model package create tests"
+    description: "Part of Model package CRD tests."
     scenarios:
-     - name: "Create=InvalidInput"
-       description: "Given one of the parameters is invalid, ko.Status shows a terminal condition"
-       given:
-         desired_state: "model_package/v1alpha1/mp_invalid_before_create.yaml"
-         svc_api:
-           - operation: CreateModelPackageWithContext
-             error:
-               code: InvalidParameterValue
-               message: "The model package name must not include a special character."
-       invoke: Create
-       expect:
-         latest_state: "model_package/v1alpha1/mp_invalid_create_attempted.yaml"
-         error: resource is in terminal condition
+      - name: "Create=InvalidInput"
+        description: "Given one of the parameters is invalid, Status shows a terminal condition"
+        given:
+          desired_state: "v1alpha1/create/desired/invalid_before_create.yaml"
+          svc_api:
+            - operation: CreateModelPackageWithContext
+              error:
+                code: InvalidParameterValue
+                message: "The Model package name must not include a special character."
+        invoke: Create
+        expect:
+          latest_state: "v1alpha1/create/observed/invalid_create_attempted.yaml"
+          error: resource is in terminal condition
+      - name: "Create=Valid"
+        description: "Create a new Model package successfully (ARN in status)."
+        given:
+          desired_state: "v1alpha1/create/desired/success_before_create.yaml"
+          svc_api:
+            - operation: CreateModelPackageWithContext
+              output_fixture: "sdkapi/create/create_success.json"
+        invoke: Create
+        expect:
+          latest_state: "v1alpha1/create/observed/success_after_create.yaml"
+          error: nil
+      - name: "Create=ValidVariation"
```

Comment:
Better than Create=Valid2 but still can we add a better name to know why a variation needs to be tested - is it more fields ?

Reply:
renamed to clarify the inputs